### DISCUSSION TITLE
Question about graphs in astronomical calculus window

### DISCUSSION THREAD

**Discussion by cbellang** [Stellarium/stellarium#1632] [Q&A] [answered]
Is it possible to save (in ascii or xls format for example) the datas from graphics in astronomical calculus window ? 

Indeed, I need to get the elevation vs time and the azimut vs time for several stars, at different days and location, to compare them and plan a future observation. Getting this datas would be a good way to perform this. 

regards,

**Comment by alex-w** [comment, marked as answer]
> Is it possible to save (in ascii or xls format for example) the datas from graphics in astronomical calculus window ?

No, because we don’t implement export.

> Indeed, I need to get the elevation vs time and the azimut vs time for several stars, at different days and location, to compare them and plan a future observation. Getting this datas would be a good way to perform this.

The best way for solution of your task will be use scripting engine to get a data.

**Reply by cbellang** [reply]
Ok ! Thank you very much for the answer. 

How do I start with the scripting engine ? (I never use it).

**Reply by alex-w** [reply]
https://stellarium.org/doc/0.21/scripting.html

**Comment by cbellang** [comment]
Thank you very much !
I got my datas by scripting. I works well.

regards,






Le 02/05/2021 à 06:09, Alexander V. Wolf a écrit :
>
>     Is it possible to save (in ascii or xls format for example) the
>     datas from graphics in astronomical calculus window ?
>
> No, because we don’t implement export.
>
>     Indeed, I need to get the elevation vs time and the azimut vs time
>     for several stars, at different days and location, to compare them
>     and plan a future observation. Getting this datas would be a good
>     way to perform this.
>
> The best way for solution of your task will be use scripting engine to 
> get a data.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Stellarium/stellarium/discussions/1632#discussioncomment-684131>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AT4WPXB4IEJBPQHZ6UPZVUDTLTGBLANCNFSM434O3QOA>.
>

-- 
*********************************
Cindy Bellanger

ONERA/DOTA/CIO
BP 80100
91123 PALAISEAU Cedex

Tel : 01.80.38.63.84

***@***.***

*********************************
